src/ORM/Association.php

```diff
@@ -893,7 +894,7 @@ public function exists($conditions): bool
      * @see \Cake\ORM\Table::updateAll()
      * @return int Count Returns the affected rows.
      */
-    public function updateAll(array $fields, $conditions): int
+    public function updateAll(array $fields, mixed $conditions): int
```

Comment:
Are the types for types for both params the same as for Table::updateAll()?

Reply:
Updated updateAll() and deleteAll().